public/themes/custom/hdbt_subtheme/src/scss/components/view/_application-search.scss

```diff
@@ -154,7 +151,7 @@
   margin-bottom: var(--spacing-l);
   position: relative;
   width: 100%;
-  border: 1px var(--color-black-20) solid;
+  box-shadow: 1px 2px 4px 0px rgba(0, 0, 0, 0.5);
```

Comment:
Mun hämähäkkivaisto sanoi, että tän pitäisi olla variable joka tulee meille parent themestä, mutta se ei ole. Tämä on siis ok, koska näitä box-shadow'ta ei ole muualla, mutta pidetään mielessä sitä varten, että jos niitä tulee muualle, että tehdään tästä siinä kohtaa variable johonkin.